src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -458,14 +456,16 @@ public void run() {
                     LOGGER.error("Could not save permissions!", e);
                 }
             }
-        }.repeat(300000, 60000);
+        }.repeat(TimeUnit.MINUTES.toMillis(5), TimeUnit.MINUTES.toMillis(10));
```

Comment:
Ok let's recap.
minute = 60 seconds
second = 1000 milliseconds
`60_000` is 60 * 1000 milliseconds
If we divide that by second -> millisecond conversion the 1000 cancels out leaving us with 60 seconds, and when we convert that with a minute to second conversion we cancel out the 60 and get the number `1`.